crates/autopilot/src/driver_model.rs

```diff
@@ -78,6 +78,8 @@ pub mod solve {
         #[serde_as(as = "Option<DisplayFromStr>")]
         pub price: Option<U256>,
         pub trusted: bool,
+        pub decimals: Option<u8>,
+        pub symbol: Option<String>,
```

Comment:
I would move this into the driver. It's not core to the protocol and some solvers don't care about that at all.

Reply:
Agree with @MartinquaXD on this one.

Reply:
I agree as well but this will take more time to do.

Reply:
Can you add a `TODO` and create a GitHub issue?